includes/KSES.php

```diff
@@ -541,6 +541,7 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 				'object-fit'                => true,
 				'object-position'           => true,
 				'placeholder'               => true,
+				'sizes'                     => true,
```

Comment:
Now need to add `disable-inline-width` here as well.

We could add a new test to `\Google\Web_Stories\Tests\KSES::data_test_filter_kses_allowed_html()` for this new image output

Reply:
Added now.